packages/app-desktop/ElectronAppWrapper.test.ts

```
@@ -0,0 +1,279 @@
+			expect(showSplash).toBe(true);
+		});
+	});
+
+	describe('destroySplashWindow idempotency', () => {
+
+		test('should safely handle null splashWindow', () => {
+			// Simulates calling destroySplashWindow when no splash was created
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Testing internal state
+			const context: any = { splashWindow_: null };
+
+			const destroySplashWindow = function(this: typeof context) {
```

ESLint

[error] 'this' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)